src/providers/authProvider/index.ts

```diff
@@ -84,11 +88,11 @@ const updateLockouAttempts = async (
   })
 }
 
-const fetchUser = async (username: string) => {
+const fetchUser = async (username: string): Promise<any> => {
   const user = await axios.get(
     `http://localhost:8000/api/tables/_users/rows?_filters=username:${username}`
```

Comment:
we can't use a hard-coded URL - it may be served on another port.

Reply:
I'm pretty sure this hard-coded path is causing the `yarn test` failure: https://github.com/DeepBlueCLtd/RCO/actions/runs/8374624709/job/22930278394

Reply:
When we wrote the test case it was fetching permissions from the frontend but now as the permissions logic is shifted to backend, therefore it doesn't find the server running to make api call to fetch permissions therefore the connection error.

Reply:
ok, I guess we can remove the test, since  the capability is no longer provided by RCO/VAL.